hotel_daily_service/wizard/hotel_daily_service_wizard.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from datetime import timedelta
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from dateutil import parser
+except (ImportError, IOError) as err:
+    _logger.debug(err)
+
+
+class HotelDailyServiceLineWizard(models.TransientModel):
+    _name = "hotel.daily.service.line.wizard"
+    _description = "Hotel Daily Service Line"
+
+    @api.model
+    def _default_hotel_service_id(self):
+        breakfast = self.env.ref("hotel_daily_service.hotel_service_breakfast")
+        return breakfast
+
+    hotel_folio_id = fields.Many2one(
+        comodel_name="hotel.folio", string="Folio"
+    )
+    hotel_service_id = fields.Many2one(
+        comodel_name="hotel.services",
+        string="Service",
+        default=_default_hotel_service_id,
+    )
+    hotel_service_line_ids = fields.Many2many(
+        comodel_name="hotel.service.line",
+        string="Service Lines",
+        relation="hotel_service_line_daily_service_wizard",
+    )
+    quantity = fields.Integer(string="Quantity", default=1)
+
+    @api.model
+    def default_get(self, field_names):
+        defaults = super().default_get(field_names)
+        defaults["hotel_folio_id"] = (
+            self.env["hotel.folio"].browse(self.env.context["active_id"]).id
```

Comment:
You could just pass the `active_id` directly.

Reply:
Make sure it is defined though. I reloaded the page from history and it crashed.